src/components/header/_macro-options.md

```diff
@@ -115,3 +117,14 @@
 | name       | string | false    | Sets the HTML `name` attribute for the `<button>`. Not valid if `url` is set.    |
 | url        | string | false    | If set, will create an HTML anchor link with the required classes and attributes |
 | attributes | object | false    | HTML attributes (for example, data attributes) to add to the button              |
+
+## SubNavigation
+
+| Name                   | Type                      | Required | Description                                                                                                                    |
+| ---------------------- | ------------------------- | -------- | ------------------------------------------------------------------------------------------------------------------------------ |
+| id                     | string                    | true     | The HTML `id` of the `<nav>` element. Used for the `aria-controls` attribute for the menu button displayed on small viewports. |
+| ariaLabel              | string                    | false    | The `aria-label` attribute added to the `<nav>` element. Defaults to “Main menu”.                                              |
```

Comment:
This doesn't default to "Main menu" when using subNavigation", its "Section menu"

Reply:
updated